java/com/google/copybara/Workflow.java

```diff
@@ -228,6 +225,18 @@ public Info getInfo() throws RepoException, ValidationException {
       return Info.create(ImmutableList.of(migrationRef));
     });
   }
+  @Nullable
+  private String maybeGetLastRev() throws RepoException, ValidationException {
+    return (getLastRevisionFlag() != null) ?
+            getLastRevisionFlag() :
+            getPreviousRefFromOrigin();
```

Comment:
Nit format and parenthesis :

return getLastRevisionFlag() != null 
    ? getLastRevisionFlag() 
    :  getPreviousRefFromOrigin();